client/my-sites/plans/current-plan/index.jsx

```diff
@@ -223,18 +186,16 @@ export default connect( ( state, { requestThankYou, requestProduct } ) => {
 	const currentPlan = getCurrentPlan( state, selectedSiteId );
 
 	return {
-		selectedSite,
-		selectedSiteId,
+		currentPlan,
 		domains,
-		currentPlan: currentPlan,
-		isExpiring: isCurrentPlanExpiring( state, selectedSiteId ),
-		isFreePlan: isSiteOnFreePlan( state, selectedSiteId ),
-		shouldShowDomainWarnings: ! isJetpack || isAutomatedTransfer,
 		hasDomainsLoaded: !! domains,
 		isRequestingSitePlans: isRequestingSitePlans( state, selectedSiteId ),
 		purchase: currentPlan ? getByPurchaseId( state, currentPlan.id ) : null,
+		requestProduct,
```

Comment:
This prop is provided directly to the component instance, so why overwrite that with a connected value? I think we can remove it.